scripts/import-vendors.ts

```
@@ -223,29 +77,36 @@ async function importCsv(filePath: string) {
 
-  const files = fs.readdirSync(vendorsDir).filter(f => f.endsWith(".csv"));
+  const files = fs.readdirSync(vendorsDir).filter((f) => f.endsWith(".csv"));
 
   if (files.length === 0) {
     console.error(`❌ No CSV files found in: ${vendorsDir}`);
     process.exit(1);
   }
 
   console.log(`Found ${files.length} CSV file(s):`);
-  files.forEach(f => console.log(`  - ${f}`));
+  files.forEach((f) => console.log(`  - ${f}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)